CRM/Contact/BAO/ContactType.php

```diff
@@ -617,8 +599,9 @@ public static function del($contactTypeId) {
 FROM civicrm_navigation
 WHERE name = %1";
       $params = [1 => ["New $name", 'String']];
-      $dao = CRM_Core_DAO::executeQuery($sql, $params);
+      CRM_Core_DAO::executeQuery($sql, $params);
       CRM_Core_BAO_Navigation::resetNavigation();
+      Civi::cache('contactTypes')->clear();
```

Comment:
why are you using `clear()` here when we have consistently used `flush()` which is an alias of clear

Reply:
because flush is actually deprecated!

Reply:
ok do we want consistance or non deprecated code paths :-P

Reply:
we get consistency when the others are migrated over